### DISCUSSION TITLE
Tiny Blazor Admin only shows Home and Help on left hand nav pane

### DISCUSSION THREAD

**Discussion by barryswales** [microsoft/AzUrlShortener#515] [Q&A]
I log in at the top right as an Administrator, but still don't see the other options I see on the screenshots.

I also get a 404 error when I test the redirect, despite setting azure.com as the redirect destination when seting up the app. Any help would be greatly appreciated.

B

**Comment by ocoinfocomm** [comment]
Try to deploy the backend again, and make sure you have entered the Static Web App name during the deployment setup.

Existing SWA Name: If you are using [TinyBlazorAdmin], this is the name of the already deployed TinyBlazorAdmin static web app.
